core/storage/statistics/gae_models.py

```diff
@@ -728,6 +717,61 @@ def create(
         return entity_id
 
 
+class StateFinishEventLogEntryModel(base_models.BaseModel):
+    """An event triggered by a student finishing a state.
+
+    Event schema documentation
+    --------------------------
+    V1:
+        event_type: 'state_finish'
+        exp_id: id of exploration currently being played
+        exp_version: version of exploration
+        state_name: Name of current state
+        client_time_spent_in_secs: Time since start of this state when this
```

Comment:
Honestly, this isn't a great name; it doesn't really convey the meaning of this field. Maybe time_spent_in_state_secs, or similar? Ditto for other models if there's a chance to improve naming and they're not live yet.

Reply:
Done, for all new models.